src/styles/utils/variables.scss

```diff
@@ -0,0 +1,14 @@
+$BoxSize: 100px;
```

Comment:
Use correct naming for variables, if you use camelCase - $boxSize.

Reply:
Fixed